snapcraft.yaml

```
@@ -555,6 +555,29 @@ parts:
       cp -r usr/share/locale-langpack $CRAFT_PART_INSTALL/usr/share/
 
 
+  poppler:
+    after: [ cairo, gdk-pixbuf, glib, gobject-introspection, gtk3, meson-deps ]
+    source: https://gitlab.freedesktop.org/poppler/poppler.git
+    source-tag: 'poppler-22.07.0'
+    source-depth: 1
+    plugin: cmake
+    cmake-parameters:
+      - -DCNAKE_INSTALL_RREFIX=/usr
```

I'm not sure whether it's needed, but we might want to ensure that Cmake builds for Release (might be default for poppler, but I didn't check)

```suggestion
      - -DCNAKE_INSTALL_RREFIX=/usr
      - -DCMAKE_BUILD_TYPE=Release
```